library/spdm_responder_lib/libspdm_rsp_finish.c

```diff
@@ -603,8 +674,15 @@ libspdm_return_t libspdm_get_response_finish(libspdm_context_t *spdm_context, si
     spdm_response->header.param1 = 0;
     spdm_response->header.param2 = 0;
 
+    ptr = (uint8_t *)spdm_response + sizeof(spdm_finish_response_t);
+    if (libspdm_get_connection_version(spdm_context) >= SPDM_MESSAGE_VERSION_14) {
```

Comment:
Why doesn't the Responder generate opaque data?

Reply:
No OpaqueData is required in FINISH by SPDM spec. It is same as today. No additional device specific OpaqueData is required in KEY_EXCHANGE.
This is for future Auth Spec.